### ISSUE TITLE
Empty commits on open PR

### ISSUE THREAD

**Issue by nagano** [fjogeleit/yaml-update-action#549]
After opening a PR with the changes, if I run the action again it pushes an empty commit to the branch. 
Is there any way to avoid this? 

I have a workflow that runs a couple times a week and the PR can remain open for a while, and I'd like to avoid the noise.

**Comment by nagano** [comment]
Correct. After the first run, it pushes a commit with no content on the next runs

**Comment by fjogeleit** [comment]
The action would stop if the content would not change but in your case its not possible.

If you create branches for your changes, you could check if a branch already exists and stop if so.

```sh
git ls-remote --exit-code --heads origin <branch-name>
```

**Comment by fjogeleit** [comment]
Hmm okay, I can check if I find a possible check to avoid it but not sure if I can do much against it.
